### DISCUSSION TITLE
cn/2024/08/20/laws/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#177] [Announcements]
# cn/2024/08/20/laws/

https://hongtaoh.com/cn/2024/08/20/laws/

**Comment by earfanfan** [comment]
>不要拿别人的名字开玩笑

完犊子了，我已经称呼你郝糊涂好多次了，你没有提出反对，我就默认你不介意。

**Reply by hongtaoh** [reply]
熟了之后就没事。如果是刚认识，就不太好了～
